src/app/modules/rup/components/ejecucion/puntoInicio.component.ts

```diff
@@ -478,7 +478,7 @@ export class PuntoInicioComponent implements OnInit, OnDestroy {
     getPrestaciones() {
         return this.servicioPrestacion.get({
             fechaDesde: this.fecha ? this.fecha : new Date(),
```

Comment:
Esté parámetro se podría asignar igual que `fechaHasta`? Es necesario el contro? En esta y otras líneas se evalúa `this.fecha`, cuando se setea en la declaración como `new Date()` y no se vuelve a cambiar en ningún lado.